.github/workflows/publishrpm.yml

```diff
@@ -0,0 +1,47 @@
+name: Publish rpm from master
+
+on:
+  schedule:
+    # every night at one
+    - cron: '0 1 * * *'
+  push:
+    branches:
+      - master
+
+env:
+  CI_IMAGE: fiware/orion-ci:rpm8
+
+jobs:
+  build-rpm-release:
```

Comment:
When did job is executed? What 'push' event actually means? Whenever a tag is created in the repository (such as 3.4.0, etc.)?

Reply:
Yes we use tags for releasing and I think it would be a good idea to change to tags.

Could you add a pattern for tags? Only tags which match the "x.y.z" (being x, y and z positive integers) should produce a RPM release.

Reply:
see: #01932c1a it should do the trick